ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -685,11 +689,18 @@ def test_ipa_healthcheck_revocation(self):
         assert returncode == 1
         assert len(data) == 12
 
+        version = tasks.get_healthcheck_version(self.master)
         for check in data:
             if check["kw"]["key"] == request_id:
                 assert check["result"] == "ERROR"
                 assert check["kw"]["revocation_reason"] == "unspecified"
-                assert check["kw"]["msg"] == error_msg
+                if version[2] > "0.4":
```

Comment:
It's safer to use parse_version() from pkg_resources to compare values. A string comparison would wrongly return that "0.3" > "0.10" for instance.

Reply:
@flo-renaud  have used parse_version() from pkg_resources now